streams-contrib/streams-provider-twitter/src/main/jsonschema/org/apache/streams/twitter/api/FavoritesListRequest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "$license": [
+    "http://www.apache.org/licenses/LICENSE-2.0"
+  ],
+  "id": "#",
+  "javaType" : "org.apache.streams.twitter.api.FavoritesListRequest",
+  "javaInterfaces": ["java.io.Serializable"],
+  "properties": {
+    "user_id": {
+      "description": "The ID of the user for whom to return results. Either an id or screen_name is required for this method.",
+      "required": false,
+      "type": "integer"
```

Comment:
i don't think so.
a) the bean contains a long and 
b) the rest api turns it into a string param before sending it.
it may be the case that whomever is coding to this interface would be better off passing in Long.parseLong(id_str) than id - but i think if the argument is always a number the sdk should model it as such.